templates/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,15 @@
+## Description
+
+<!--- Describe your changes in detail -->
+
+Fixes: <!--- add github link -->
```

Comment:
Does Github work with the ':' ?

Reply:
It does, there is an example (well for "Closes") in https://help.github.com/articles/closing-issues-using-keywords/